micro/client_test.go

```diff
@@ -0,0 +1,52 @@
+package micro
+
+import (
+    "testing"
+)
+
+import (
+    "github.com/AlexStocks/getty/rpc"
+    "github.com/stretchr/testify/assert"
+)
+
+var (
+    client *Client
+    ClientConfig *rpc.ClientConfig
+    ConsumerConfig *ConsumerRegistryConfig
+)
+
+func initClientConf() {
+    ClientConfig = &rpc.ClientConfig{}
```

Comment:
pls init config in this way:


func buildClientConfig() *ClientConfig {
	return &ClientConfig{
		AppName:         "mq",
		Host:            "127.0.0.1",
		ProfilePort:     38001,
		ConnectionNum:   1,
		HeartbeatPeriod: "30s",
		SessionTimeout:  "300s",
		FailFastTimeout: "3s",
		PoolSize:        64,
		PoolTTL:         600,
		GettySessionParam: GettySessionParam{
			CompressEncoding: false,
			TcpNoDelay:       true,
			TcpKeepAlive:     true,
			KeepAlivePeriod:  "180s",
			TcpRBufSize:      262144,
			TcpWBufSize:      65536,
			PkgRQSize:        0,
			PkgWQSize:        512,
			TcpReadTimeout:   "1s",
			TcpWriteTimeout:  "3s",
			WaitTimeout:      "1s",
			MaxMsgLen:        8388608,
			SessionName:      "mq-rcp-client",
		},
	}
}


func buildServerConfig() *ServerConfig {
	return &ServerConfig{
		AppName:         "RPC-SERVER",
		Host:            ServerHost,
		Ports:           []string{ServerPort},
		ProfilePort:     10086,
		SessionTimeout:  "180s",
		sessionTimeout:  time.Second * 180,
		SessionNumber:   1,
		FailFastTimeout: "3s",
		failFastTimeout: time.Second * 3,
		GettySessionParam: GettySessionParam{
			CompressEncoding: false,
			TcpNoDelay:       true,
			TcpKeepAlive:     true,
			KeepAlivePeriod:  "120s",
			keepAlivePeriod:  time.Second * 120,
			TcpRBufSize:      262144,
			TcpWBufSize:      524288,
			PkgRQSize:        1024,
			PkgWQSize:        512,
			TcpReadTimeout:   "1s",
			tcpReadTimeout:   time.Second * 1,
			TcpWriteTimeout:  "3s",
			tcpWriteTimeout:  time.Second * 3,
			WaitTimeout:      "1s",
			waitTimeout:      time.Second * 1,
			MaxMsgLen:        102400,
			SessionName:      "getty-rpc-server",
		},
	}
}

Reply:
ok